src/pages/apply/ApplyOptions.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+
+interface ApplicationTextProps {
+  title: string;
+  body: string;
+  linkText: string;
+  url: string;
+}
+
+const ApplyOptions = ({ title, body, linkText, url }: ApplicationTextProps) => {
+  const navigate = useNavigate();
+  const handleClick = () => navigate(`/${url}`);
+
+  return (
+    <Container onClick={handleClick}>
+      <Title>{title}</Title>
+      <Body>{body}</Body>
+      <InterestedText>{linkText}</InterestedText>
+    </Container>
+  );
+};
+
+const Container = styled.div<{ last?: boolean }>`
+  width: 400px;
+  flex-grow: ${(props) => (props.last ? '0' : '1')};
```

Comment:
i think last can be removed here if it's not being passed as a prop

Reply:
Done! 🥳